kubernetes/yamls/susi-server/susi-deployment.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Deployment of the SUSI Server
+#
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: susi-server
+  namespace: web
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: susi-server
+    spec:
+      nodeSelector:
+        server: primary
+      containers:
+      - name: susi-server
+        image: fossasia/susi_server:latest-dev
```

Comment:
@niranjan94 What does this image denote? Does it act as a starting image? Should this be `fossasia/susi_server:latest-development` or `fossasia/susi_server:latest-master` or any one of them is fine?